packages/herm-js-document/modules/index.js

```diff
@@ -88,16 +88,19 @@ class HermDoc {
     }
 
     result.content = Object.assign([], this.content, newDoc.content);
-    
+
     return result;
   }
 
-  deleteRange(from, to) {
-  	let r = to - from;
-    for(let i = 0 ; i < r; i++) {
-    	const k = this.keys[from + i];
-      this.content[k] =  null;
+  deleteRange(from, length) {
+    for(let i = 0; i < length; i++) {
+      const k = this.keys[from + i];
+
+      delete this.content[k];
     }
+
+    this.keys = [...this.keys];
+    this.keys.splice(from, length);
```

Comment:
I think we shouldn't remove them from the indexes : they help with merging outdated documents

Reply:
It's up to you. I did this "simplistic" fix just to pass tests.

Reply:
Please review https://github.com/azukaar/herm-js/pull/2 and then rebase, this will fix the issue you were having with the function